docs/raw/push-notification-server.md

```diff
@@ -514,6 +518,9 @@ Any chat id in this list will not trigger a notification.
 `unregister`: whether the account should be unregistered
 `grant`: the grant for this specific server
 `allow_from_contacts_only`: whether the client only wants push notifications from contacts
+`apn_topic`: the APN topic for the push notification
+`block_mentions`: whether the client does not want to be notified on mentions
```

Comment:
How does `block_mentions` interact with `allow_from_contacts_only`?

Reply:
if `allow_from_contacts_only` is only, only mentions from contacts will be shown